### DISCUSSION TITLE
Although in the new beta, new repos have the old actions

### DISCUSSION THREAD

**Discussion by dennisnewel** [community/community#25890] [Actions]
<p>I’ve successfully setup the new actions for one repo, but now that i want to do it for another repo, i don’t see a way to “upgrade” it or “enable” the new actions. I honestly can’t remember how i got the first repo enabled, but i would like to be able to use it with multiple repos </p>

**Comment by judge2020** [comment]
<p>When I create a new repo, the actions is already upgraded to the new system. Have you tried creating the .github/workflows/&lt;something&gt;.yml file to see if it works?</p>

**Comment by dennisnewel** [comment]
<p>I have not tried that yet. good point though :slight_smile: i’ll give that a shot</p>

**Comment by dennisnewel** [comment]
<p>no luck…it’s a public repo, but making it private doesn’t seem to make a difference…</p>
<p>I was able to upgrade another repo that had an old workflow file, so it seems to still be available at least…</p>

**Comment by mscoutermarsh** [comment, marked as answer]
<p>Hi Dennis,</p>
<p>I could upgrade your entire account. Sound good?</p>

**Comment by dennisnewel** [comment]
<p>that would be awesome! thanks <a class="mention" href="/mscoutermarsh">@mscoutermarsh</a> </p>

**Comment by mscoutermarsh** [comment]
<p>Done. try now. :sunglasses:</p>

**Comment by dennisnewel** [comment]
<p>success!! thank you!</p>
<p>off to building docker images now :D </p>

**Comment by softprops** [comment]
<p><a class="mention" href="/mscoutermarsh">@mscoutermarsh</a> I’m having the exact same issue. I have been enrolled in both the old and new betas and have a new repo that seems to insist on only using the old workflow syntax. Adding a new style workflow file <a href="https://github.com/softprops/rust-bin/blob/master/.github/workflows/main.yml" rel="nofollow noopener">https://github.com/softprops/rust-bin/blob/master/.github/workflows/main.yml</a> in place doesn’t seem to be detected by the actions tab.</p>
<p>For some extra context in other repos there’s usually a prompt (that I don’t need) to opt into the new syntax. This repo didn’t have it which makes this repo unique. If possible can I only be enrolled in the new syntax beta to avoid new repos using the old syntax.</p>

**Comment by mscoutermarsh** [comment]
<p><a class="mention" href="/softprops">@softprops</a> Just enabled for your entire account. Give it a try now :grinning:</p>

**Comment by filliph** [comment]
<p>Same issue for me, I was only able to upgrade a single repo - I have quite a number of them and I’d like to switch them all from webhooks to actions :smile:</p>

**Comment by alfg** [comment]
<p>Also having the same issue. I would like to use the new actions in my other repos as well.</p>

**Comment by softprops** [comment]
<p><a class="mention" href="/mscoutermarsh">@mscoutermarsh</a> you are the best! Thanks for the fast turn around.</p>

**Comment by ethomson** [comment]
<p><a class="mention" href="/alfg">@alfg</a>  and <a class="mention" href="/filliph">@filliph</a> , I’ve enabled your user accounts.</p>

**Comment by raj-al-ghul** [comment]
<p>seems like I am having the same problem</p>

**Comment by rdmurphy** [comment]
<p>I believe I’m seeing the same thing as well. New repos appear to have new workflows, but all of my old ones do not. An organization I’m in also just got access to the beta, and even its older repos are using the new style, so perhaps I’m in a weird limbo too! :grimacing:</p>

**Comment by chrisns** [comment]
<p>+1 any chance of having my whole <a class="mention" href="/chrisns">@chrisns</a> account done so my new repos can use new actions?</p>

**Comment by hakusaro** [comment]
<p>If this happens to anyone else, you can get around it by creating an HCL syntax workflow. Then you’ll see the option to upgrade. You need an old-style one and then it will just offer it and you can move forward without waiting for staff to upgrade you manually.</p>
